etc/cowrie.cfg.dist

```diff
@@ -1048,3 +1048,16 @@ bearer_token = THREATJAMMER_API_TOKEN
 [output_discord]
 enabled = false
 url = https://discord.com/api/webhooks/id/token
+
+# DataDog output module
+# sends JSON directly to DataDog
+# mandatory field: api_key
+# optional fields (fallback configured in module): ddsource, ddtags, service
+# For more information on fields https://docs.datadoghq.com/api/latest/logs/#send-logs
+[output_datadog]
+enabled = false
+url = https://http-intake.logs.datadoghq.com/api/v2/logs
```

Comment:
I believe this URL needs to be adapted if you use a non-default region? (e.g. EU or GOV) If yes let's add it to the docs

Reply:
From the [official API documentation](https://docs.datadoghq.com/api/latest/logs/), there's only one mentioned endpoint.

Reply:
If you change your region in the top right corner the URL changes to e.g. `https://http-intake.logs.datadoghq.eu/api/v2/logs` so I think it's worth at least a mention

![image](https://user-images.githubusercontent.com/136675/201932717-c13512f6-dcb9-41a6-974a-d24fb964931b.png)